web_widget_digitized_signature/__manifest__.py

```diff
@@ -1,18 +1,20 @@
 # Copyright 2004-2010 OpenERP SA (<http://www.openerp.com>)
 # Copyright 2011-2015 Serpent Consulting Services Pvt. Ltd.
 # Copyright 2017 Tecnativa - Vicent Cubells
+# Copyright (C) 2019 - TODAY, Open Source Integrators
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {
-    "name": "Web Widget Digitized Signature",
-    "version": "11.0.1.0.0",
-    "author": "Serpent Consulting Services Pvt. Ltd., "
-              "Agile Business Group, "
-              "Tecnativa, "
-              "Odoo Community Association (OCA)",
+    'name': 'Web Widget Digitized Signature',
+    'version': '12.0.1.0.0',
+    'author': 'Serpent Consulting Services Pvt. Ltd., '
+              'Agile Business Group, '
+              'Tecnativa, '
+              'Open Source Integrators,'
+              'Odoo Community Association (OCA)',
     'website': 'https://github.com/OCA/web',
-    "license": "AGPL-3",
-    "category": 'Web',
+    'license': 'AGPL-3',
```

Comment:
Don't increase the diff if not needed. Disable automatic formatting like `black` for this.

Reply:
@pedrobaeza I think migration is a great opportunity to clean up those inconsistencies of having simple and double quote within the same manifest file.

Reply:
Well, I don't see them as inconsistencies. Both are allowed in Python the same and it doesn't produce any visual problem. We don't have any guideline that forces to unify them. Even if so, that must be done on a separate commit `[REF] style` or similar.